docs/filesystems-file-transfer/transferring-files.md

```diff
@@ -0,0 +1,237 @@
+# Transferring Files
+
+There are a few different ways to transfer files depending on your goals, the data you are transferring, and what you are comfortable with. On this page we cover the different methods of transferring files, as well as touch on how to transfer files between systems.
+
+For most of these options you will need to know the hostname of the node where you will be transferring files. This is often a login node, but may also be a dedicated data transfer node. Select the system you are using to see options for the hostname here:
+
+=== "Engaging"
+
+    - `eofe7.mit.edu`
+    - `eofe8.mit.edu`
+    - `eofe9.mit.edu`
+    - `eofe10.mit.edu`
+
+=== "Satori"
+
+    - `satori-login-001.mit.edu`
+    - `satori-login-002.mit.edu`
+
+=== "OpenMind"
+
+    - `openmind-dtn.mit.edu`
+
+=== "SuperCloud"
+
+    - `txe1-login.mit.edu`
+
+For more information specific to the system you are using, you can consult your system's documentation here:
```

Comment:
Might be more clear if you could include that documentation is not available for Engaging. If it was not included, I would not know whether if it didn't exist or you forgot to put it in.

Reply:
Added a note "Engaging does not have an additional documentation page."